lib/vrf.h

```diff
@@ -111,15 +111,24 @@ extern struct vrf *vrf_get(vrf_id_t, const char *);
 extern const char *vrf_id_to_name(vrf_id_t vrf_id);
 extern vrf_id_t vrf_name_to_id(const char *);
 
-#define VRF_GET_ID(V, NAME)                                                    \
+#define VRF_GET_ID(V, NAME, USE_JSON)                                          \
```

Comment:
yikes - how did we end up with a macro which calls vtyout without any indication in the name or passing vty as argument.  This is nasty!  

of course this is *not* a comment on this PR...